tests/integration/cartography/intel/aws/ec2/test_ec2_snapshots.py

```diff
@@ -1,11 +1,46 @@
 import cartography.intel.aws.ec2.snapshots
+import cartography.intel.aws.ec2.volumes
 import tests.data.aws.ec2.snapshots
+import tests.data.aws.ec2.volumes
 
 TEST_ACCOUNT_ID = '000000000000'
 TEST_REGION = 'eu-west-1'
 TEST_UPDATE_TAG = 123456789
 
 
+def test_get_snapshots_in_use(neo4j_session):
+    # Arrange
+    data = tests.data.aws.ec2.volumes.DESCRIBE_VOLUMES
+    transformed_data = cartography.intel.aws.ec2.volumes.transform_volumes(data, TEST_REGION, TEST_ACCOUNT_ID)
+
+    # Act
+    neo4j_session.run(
+        """
+        MERGE (aws:AWSAccount{id: {aws_account_id}})
+        ON CREATE SET aws.firstseen = timestamp()
+        SET aws.lastupdated = {aws_update_tag}
+        """,
+        aws_account_id=TEST_ACCOUNT_ID,
+        aws_update_tag=TEST_UPDATE_TAG,
+    )
+    cartography.intel.aws.ec2.volumes.load_volumes(
+        neo4j_session,
+        transformed_data,
+        TEST_REGION,
+        TEST_ACCOUNT_ID,
+        TEST_UPDATE_TAG,
+    )
+
+    # Assert
+    expected_snapshots = [
+        "sn-01", "sn-02",
+    ]
+
+    actual_snapshots = cartography.intel.aws.ec2.snapshots.get_snapshots_in_use(neo4j_session, TEST_REGION, TEST_ACCOUNT_ID)
```

Comment:
One last thing for the linter
```suggestion
    actual_snapshots = cartography.intel.aws.ec2.snapshots.get_snapshots_in_use(
        neo4j_session, 
        TEST_REGION, TEST_ACCOUNT_ID
    )
```